README.md

```diff
@@ -2,7 +2,7 @@ Attribution
 -----------
 
 This library is a fork of https://github.com/aloiscochard/grpc-haskell that we
-have extended and released under the same [`LICENSE`](./LICENSE)
+have extended and released under the same [`LICENSE`](./LICENSE). It's not ready for production use and use it with caution.
```

Comment:
We do use this project quite a bit in production, so I do think it is production ready. However, we have not polished the project, it lacks good documentation, and some dependencies have fallen far behind so I think it is accurate to say:

```suggestion
have extended and released under the same [`LICENSE`](./LICENSE). Please use with caution in production.
```